comms/dht/src/crypt.rs

```diff
@@ -66,6 +69,49 @@ pub fn generate_ecdh_secret(secret_key: &CommsSecretKey, public_key: &CommsPubli
     output
 }
 
+fn pad_message_to_base_length_multiple(message: &[u8]) -> Vec<u8> {
+    let n = message.len();
+    // little endian representation of message length, to be appended to padded message,
+    // assuming our code runs on 64-bits system
+    let prepend_to_message = (n as u64).to_le_bytes();
```

Comment:
The maximum message size permitted by the protocol is 4MiB, so using u32 is plenty

Reply:
Done